apps/web/src/components/Settings/Profile/Profile.tsx

```diff
@@ -172,7 +172,7 @@ const ProfileSettingsForm: FC<ProfileSettingsFormProps> = ({ profile }) => {
       const id = await uploadToArweave({
         name,
         bio,
-        cover_picture: cover ? cover : null,
+        cover_picture: coverIpfsUrl ? coverIpfsUrl : null,
```

Comment:
will `coverIpfsUrl ?? null` works?

Reply:
Not as a one-line change, because, `??` checks for nullish values, but `?` checks for "falsy" values. So they will work differently for the empty string `''`, which is the default value for `coverIpfsUrl`.
We could of course change to:
`const [coverIpfsUrl, setCoverIpfsUrl] = useState<string | null>(null);` (and from `coverIpfsUrl` to `coverIpfsUrl ?? ''` wherever it's used) but I don't know if it's better?